lib/open_food_network/orders_and_fulfillments_report.rb

```diff
@@ -267,5 +279,18 @@ def total_units(line_items)
       end
       total_units.round(3)
     end
+
+    # Returns the appropriate shipping method to display for the given order
+    #
+    # @param order [Spree::Order]
+    # @return [#name]
+    def shipping_method_to_display(order)
+      shipment = order.shipments.last
+      if shipment
+        shipment.shipping_method
+      else
+        NullShippingMethod.new
+      end
+    end
```

Comment:
We'll need to wait for @oeoeaio 's comment https://github.com/openfoodfoundation/openfoodnetwork/pull/1659#issuecomment-314685297. We might not need it.